addons/web/static/src/views/pivot/pivot_controller.js

```diff
@@ -16,6 +16,7 @@ export class PivotController extends Component {
         Model: Function,
         modelParams: Object,
         Renderer: Function,
+        buttonTemplate: { type: String, optional: true },
```

Comment:
I'd set it mandatory, s.t. we don't duplicate the default fallback

Reply:
Indeed, I agree, I will do it. 🙂